### PR TITLE
[KOGITO-4340] - Create Go module for community kogito operator

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

Jira Issue : https://issues.redhat.com/browse/KOGITO-4340

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster
- [x] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -434,20 +434,20 @@ API server listening at: [::]:2345
 **Create the Go Remote run/debug configuration:**
 
 1. Click `Edit | Run Configurations`. Alternatively, click the list of run/debug configurations on the toolbar and select `Edit Configurations`.  
-![alt text](./docs/images/add_configuration.png?raw=true)
+![alt text](docsmages/add_configuration.png?raw=true)
```
Comment (sutaakar) [inline README.md:437]:
```suggestion
![alt text](./community-kogito-operator/docs/images/add_configuration.png?raw=true)
```

File: README.md [hunk 2]
```diff
@@ -434,20 +434,20 @@ API server listening at: [::]:2345
...
 2. In the `Run/Debug Configurations` dialog, click the `Add` button (`+`) and select `Go Remote`.
-![alt text](./docs/images/add_go_remote_config.png?raw=true)
+![alt text](docsmages/add_go_remote_config.png?raw=true)
```
Comment (sutaakar) [inline README.md:439]:
```suggestion
![alt text](./community-kogito-operator/docs/images/add_go_remote_config.png?raw=true)
```

File: README.md [hunk 3]
```diff
@@ -434,20 +434,20 @@ API server listening at: [::]:2345
...
 3. In the Host field, type the host IP address (in our case `localhost`).
 4. In the Port field, type the debugger port that you configured in above `dlv` command (in our case it's `2345`).
 5. Click `OK`.          
-![alt text](./docs/images/remote_debug_configurations.png?raw=true)
+![alt text](docsmages/remote_debug_configurations.png?raw=true)
```
Comment (sutaakar) [inline README.md:443]:
```suggestion
![alt text](./community-kogito-operator/docs/images/remote_debug_configurations.png?raw=true)
```

**Comment by sutaakar** [comment]
I suppose all the files are just moved to community-kogito-operator folder and import statements adjusted, right? So there is no need to dig into the code.
I think that we could split the readme, moving relevant parts to community-kogito-operator folder.

**Comment by vaibhavjainwiz** [comment]
> I suppose all the files are just moved to community-kogito-operator folder and import statements adjusted, right? So there is no need to dig into the code.
> I think that we could split the readme, moving relevant parts to community-kogito-operator folder.

@sutaakar Yes Karel, No cade has been changed. I just dragged files from one folder to another and fix import path

File: README.md (hunk 1 above)
Comment (vaibhavjainwiz) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (vaibhavjainwiz) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (vaibhavjainwiz) [reply README.md]: done

**Review by ricardozanini — COMMENTED** [review]
Are we ok with `community-kogito-operator` for the module name? I believe we can bring this to Thu meeting before merging.

The other thing I noticed, is that git is marking some files as `deleted` and others as `created`. This way we are losing contributions from everyone who authored these files. Is there a way to keep them in `moved` state instead?

**Comment by vaibhavjainwiz** [comment]
As discussed over Gchat channel, this PR is no more required. community kogito operator will be represented by this repo where as product kogito operator will be represented by rhpam-kogito-operator repo(https://github.com/kiegroup/rhpam-kogito-operator/).

**Pull request closed without merging**